src/main/markdown/meetingnotes.md

```diff
@@ -1,137 +1,16 @@
-* GWT 2.8:
-    * Main missing feature is JsInterop
-        * Expectation is that this will be complete by end of Sept.
-    * Plan to release after JsInterop is done. Will accept Java8 patches from community, but those are not blockers for right now. 
-* Update on new compiler prototype:
-    * Still working through compilation of Java features.
-    * Plan is to have it capable of building complete apps by the end of the year.
-    
-## 06/17/2015 
-
-Agenda:
-
-* 2.8 and follow-up
-* Issues list
-
-2.8 and follow-up:
-
-* See meet-up vidoes for details.
-* GWT 2.8: 90% done. mainly waiting for Interop to finish
-    * Christian: waiting for jsinterop bug fixes
-    * GSS migration done for all of Google apps
-        * Fixed a few small bugs
-        * External folks can use a tool to convert CSS files to GSS: need to finish up this tool.
-    * Java8 emulation patches - need to get these in
-        * Bhaskar to ping reviewers to get these reviewed
-* Google internally will sync to 2.8 branch after 2.8 is released, which will be used by internal apps that are not moving to the new tool.
-    * See Daniel’s presentation on ‘Modernizing GWT Applications’ for the migration story.
-* GWT 2.8 will be a long-lived maintenance branch
-* Master will be open for checkins, vetted by GWT Maintainers. 
-* Christian to look into writing a blog post.
-
-Issues List:
-
-* Issues moved to github. Downloads moved to Google Compute Engine.
-    * Daniel plans to write a summary of what changed and send it to contrib.
-* Thomas: Life of an issue on github
-    * No direct mapping for closed but not fixed.
-    * Few other issues
-    * Thomas: plan to write up a short policy for issues and circulate with SC.
-    
-## 02/11/2015
-
-* GWT 2.8 release
-    * Manuel and Julien will do the release. Julien will be the point person.
-    * Colin also volunteered to help
-    * Features:
-        * GSS as default, new migration guide
-            * Compiler will error if CSS is used, need to use special flag for CSS
-            * Next release beyond 2.8 will not support CSS
-        * Java8 - full support for emulation is a question.
-        * Interop v1 - Google GWT engineers are working on deciding what should be included in v1.
-        * Unlikely to happen before April
-* Eclipse plug-ins for GWT
-    * GPE is still not buildable externally
-    * Plan is to super-cede GPE with GWT plug-in
-    * Both plug-ins should work side-by-side
-    * Why is GWT GPE plug-in different from other plug-ins? Should it be externally hosted similar to maven and gradle?
-    * Decision on this deferred.
+* <a href="meetingnotes/2017-05-10.html">10 May 2017</a>
+* <a href="meetingnotes/2017-04-12.html">12 April 2017</a>
+* <a href="meetingnotes/2017-03-08.html">08 March 2017</a>
+* <a href="meetingnotes/2017-02-08.html">08 February 2017</a>
+* <a href="meetingnotes/2017-01-11.html">11 January 2017</a>
+* <a href="meetingnotes/2016-07-20.html">20 July 2016</a>
```

Comment:
Should we add a "to be published" or "…" line here to at least hint that there have been meetings in this period?